charts/seed-monitoring/charts/core/charts/prometheus/values.yaml

```diff
@@ -138,6 +138,7 @@ allowedMetrics:
   nodeExporter:
   - node_boot_time_seconds
   - node_cpu_seconds_total
+  - node_disk_written_bytes_total
```

Comment:
Shall we allow list some of the other metrics as well? e.g. `node_disk_read_bytes_total` for consistency?

A more complete list of the metrics exposed by the diskstats collector.

https://github.com/prometheus/node_exporter/blob/98889b37056d02564c60d1a60ad90d53aa2a75b6/collector/diskstats_linux_test.go#L53-L278

https://github.com/prometheus/node_exporter/blob/98889b37056d02564c60d1a60ad90d53aa2a75b6/collector/diskstats_common.go#L50

https://github.com/prometheus/node_exporter/blob/98889b37056d02564c60d1a60ad90d53aa2a75b6/collector/diskstats_linux.go#L82

Note that for each attached disk device, some metrics are exposed. It is hard to estimate without trying it if it might cause a noticable overhead regarding the node-exporter resource usage (should be addressed by vpa).